homeassistant/components/nederlandse_spoorwegen/sensor.py

```diff
@@ -191,7 +191,9 @@ def device_state_attributes(self):
             attributes["arrival_delay"] = True
 
         # Next attributes
-        if self._trips[1].departure_time_actual is not None:
+        if len(self._trips) < 2:
+            attributes["next"] = "-"
```

Comment:
Of the value is missing we shouldn't add the item at all to the state attributes. That's our standard.

Reply:
Thanks, I was unaware of this. Fixed it.